modules/sdk-coin-tia/src/lib/transactionBuilderFactory.ts

```diff
@@ -0,0 +1,75 @@
+import { BaseTransactionBuilderFactory, InvalidTransactionError, TransactionType } from '@bitgo/sdk-core';
+import { TiaTransactionBuilder } from './transactionBuilder';
+import { TiaTransferBuilder } from './transferBuilder';
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { TiaTransaction } from './transaction';
+import { StakingActivateBuilder } from './StakingActivateBuilder';
+import { StakingDeactivateBuilder } from './StakingDeactivateBuilder';
+import { StakingWithdrawRewardsBuilder } from './StakingWithdrawRewardsBuilder';
+
+export class TransactionBuilderFactory extends BaseTransactionBuilderFactory {
```

Comment:
This entire class should extend from `CosmosTransactionBuilderFactory`; we should have one if we don't already